content/docs/1.7.2/deploy/install/_index.md

```diff
@@ -156,6 +156,7 @@ Longhorn components require root access with privileged permissions to achieve v
 Below are the directories Longhorn components requiring access with root and privileged permissions :
 
 - Longhorn Manager
+  - /boot: Get required modules' information from /boot/config-$(uname -r) on the host.
```

Comment:
Please make sure the mount is read-only in the implementation.

It seems more host mounts are required for environment check purposes. We need to ensure the least privilege.

Reply:
It seems we are adding similar capacities of CLI preflight to the longhorn-manager. Will this cause duplicated implementation? Why not just use CLI directly?

Reply:
CLI preflight check is usually executed before installing Longhorn system, and longhorn-manager node.status.condition is for runtime self-diagnosis. Although they check the same kernel modules and utilities, they serve different purposes.

Reply:
WDYT? @innobead @c3y1huang

Reply:
This has been discussed with @derekbit . What I care about is only duplicated implementation, so if we can achieve that in the same shared code eventually, that will be good.

Reply:
https://github.com/longhorn/longhorn/issues/9933